src/error_messages/debug_messages.h

```diff
@@ -93,6 +93,7 @@
 #define VU_UNS_OS             "(5485): Agent '%.3d' has an unsupported OS: '%s'"
 #define VU_PACKAGE_TP_SOURCE  "(5486): Discarded package '%s' from a third-party source ('%s') for agent '%.3d'"
 #define VU_ERROR_CMP_VER      "(5487): Unknown relation '%s' between versions '%s' and '%s' for package '%s'"
+#define VU_DISARD_CVE_ENTRY   "(5488): Discarded false positive: Package '%s' not affected by '%s' with misleading condition (feed '%s')."
```

Comment:
I wouldn't call we are discarding a false positive in this stage because we are not reported the package as vulnerable yet. We are simply discarding the vulnerability, so I think is better to say:
```
#define VU_DISARD_CVE_ENTRY "(5488): Package '%s' not affected by '%s' with misleading condition (feed '%s')"